queries/mustache/highlights.scm

```diff
@@ -0,0 +1,17 @@
+[
+  (start_delimiter)
+  (end_delimiter)
+] @tag.delimiter
```

Comment:
Also `"{"` and `"}"` for things like `}}}`. Also should this be `@punctuation.bracket`?

Reply:
> Also "{" and "}" for things like }}}

You are right. I'll add it

> should this be @punctuation.bracket?

I have no strong opinions about this. I could change it